float32 tree_height
float32 tree_width
float32 tree_volume
int8 treeID
